sdk/storage/azure-storage-file-share/README.md

```diff
@@ -167,21 +167,21 @@ The URI to reference a share, directory or file must be unique. Within a given s
 If you attempt to create a share, directory, or file with a name that violates naming rules, the request will fail with status code 400 (Bad Request).
 
 ### Share Names
-The rules for File service share names are more restrictive than what is prescribed by the SMB protocol for SMB share names, so that the Blob and File services can share similar naming conventions for containers and shares. The naming restrictions for shares are as follows:
+The rules for File Share service share names are more restrictive than what is prescribed by the SMB protocol for SMB share names, so that the Blob and File services can share similar naming conventions for containers and shares. The naming restrictions for shares are as follows:
 
 1. A share name must be a valid DNS name.
 1. Share names must start with a letter or number, and can contain only letters, numbers, and the dash (-) character.
 1. Every dash (-) character must be immediately preceded and followed by a letter or number; consecutive dashes are not permitted in share names.
 1. All letters in a share name must be lowercase.
 1. Share names must be from 3 through 63 characters long.
 
-### Directory and File Names
-The Azure File service naming rules for directory and file names are as follows:
+### Share Directory and File Names
```

Comment:
Either change Share to File Share or just revert this change

Reply:
Revert it back as some of other places refer to directory and file only.